homeassistant/components/goalzero/translations/en.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "already onfigured"
+        },
+        "error": {
+            "cannot_connect": "cannot connect",
+            "connect_timeout": "connect timeout",
+            "host_required": "host required",
+            "invalid_host": "invalid host",
+            "unknown": "unknown error"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "host": "host",
+                    "name": "name"
+                },
+                "title": "Goal Zero Yeti"
+            }
+        }
+    },
+    "title": "Goal Zero Yeti"
+}
```

Comment:
Do we need this? I forgot to leave it out of the commit.

Reply:
Copy strings.json to en.json.  When its merged, lokalise will take care of this.